packages/calcite-components/src/components/flow/flow.e2e.ts

```diff
@@ -55,6 +56,24 @@ describe("calcite-flow", () => {
       expect(flowItem).toBeNull();
     });
 
+    it("should call setFocus() on back button click", async () => {
+      const page = await newE2EPage();
+
+      await page.setContent(
+        html`<calcite-flow
+          ><calcite-flow-item id="one"></calcite-flow-item><calcite-flow-item id="two"></calcite-flow-item
+        ></calcite-flow>`
+      );
+
+      await page.$eval("#two", (elm: HTMLCalciteFlowItemElement) =>
```

Comment:
In order to remove implementation details, can we click the back button vs firing the custom event?

Reply:
yup!